compose.yaml

```diff
@@ -48,7 +47,7 @@ services:
         params_file:=/params.yaml
         map:=/maps/map.yaml
         use_sim_time:=False
-  
+
```

Comment:
I see that there is an old approach here that separates the navigation process from slam, in the new approach everything is in one composie, so some processes from nav2 are not duplicated (it runs smoother, there is a drop of about 15% on 4 CPU Rpi cores)

I recommend getting inspiration from here: https://github.com/husarion/rosbot-xl-autonomy/blob/ros2router/compose.yaml